target/linux/ath79/dts/qca9563_tplink_eap245-v1.dts

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca9563_tplink_eap2x5-1port.dtsi"
+
+/ {
+	compatible = "tplink,eap245-v1", "qca,qca9563";
+	model = "TP-Link EAP245 v1";
+
+	aliases {
+		led-boot = &led_status_green;
+		led-failsafe = &led_status_amber;
+		led-running = &led_status_green;
+		led-upgrade = &led_status_amber;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_status_green: status_green {
+			label = "tp-link:green:status";
```

Comment:
Please drop the tp-link: prefix for all LEDs, and just keep color:function ...

Reply:
Dropped the prefixes for the LEDs for all three devices. Will you implement a similar policy for gpio-export? Upstream still appears to have just a label ("line-name") for gpio-hog.

Reply:
I've asked myself the same question when I looked at these patches. I have not really thought about that yet.
In any case, since gpio-export is "our" solution, we should be fine to do what we want anyway. I have no idea how gpio-export interfaces with LuCI yet, this needs to be checked as well.

Reply:
Err, is gpio-export exposed to LuCI at all?

Reply:
Personally, I'd say drop the prefix for gpio-export as well for _new_ devices as these here.
BTW, nitpicking, I wonder whether it should be led:enable or leds:enable

Reply:
It's a global toggle, so I renamed "tp-link:led:enable" to "leds:enable".